cert/lib/cert/runner.rb

```diff
@@ -123,9 +123,17 @@ def certificates
 
     # The kind of certificate we're interested in
     def certificate_type
-      cert_type = Spaceship.certificate.production
-      cert_type = Spaceship.certificate.in_house if Spaceship.client.in_house?
-      cert_type = Spaceship.certificate.development if Cert.config[:development]
+      case Cert.config[:platform].to_s
+      when 'ios', 'tvos'
```

Comment:
Is it possible for this value to be 'tvos'? It looks like it is restricted to macos or ios [by this code](https://github.com/fastlane/fastlane/pull/7730/files#diff-7c141e2c870b0117f0dba21896620e39R71).